refiner-lib/src/utils.rs

```diff
@@ -5,8 +5,6 @@ use rlp::{DecoderError, Rlp};
 use sha3::Digest;
 use std::convert::TryFrom;
 
-const U64_MAX: U256 = U256([u64::MAX, 0, 0, 0]);
-
 pub fn keccak256(input: &[u8]) -> H256 {
```

Comment:
The nitpick is not related to the PR, but the same function exists in `refiner-types/src/utils.rs`. This version could be deleted.

Reply:
Good catch. Duplication removed in https://github.com/aurora-is-near/borealis-engine-lib/pull/37/commits/2aaa6f781c1a7534a1911c98d22874813543b413 . I still included it as a public import to avoid noise elsewhere in the diff.